src/view/home.js

```diff
@@ -1,87 +1,105 @@
-              <input class="progress-bar__add-input" id="addVal${number}" type="number" value="1">
-              <span class="progress-bar__add-button" id="add${number}"> +</span>
-          </div>
-        </div>
-        <a class="skill__experience exp${number}">-1/-1</a>
-    </div>
-    `);
-  }
+    _addProgressBar() {
+      const barWrapper = this._appendNewElement(
+        'div', {"class": "progress-bar__wrapper"}, '');
+      const container = this._createElement(
+        'span', {"class": "progress-bar__container"}, '', barWrapper);
+      const percent = Math.floor(
+        100 * this.model.expInThisLevel / this.model.expTillNextLevel);
 
-  LifeGamification.home.handleSkillButtons = function () {
-    const update_exp = function (skillNr) {
-      const addedExp = parseInt($("#addVal" + skillNr).val());
-      $("#addVal" + skillNr).val('1');
-      const skill = LifeGamification.skillsView[skillNr];
-      LifeGamification.models.updateExp(skill, addedExp)
-        .then(LifeGamification.home.viewLevelAndExp);
+      this._createElement(
+        'span', {
+          "class": "progress-bar__container-fill",
+          "style": `width: ${percent}%`
+        },
+        `${percent}%`, container
+      );
+
+      const pg_buttons = this._createElement(
+        'div', {"class": "progress-bar__buttons"}, '', barWrapper);
+      this._createInput(pg_buttons);
     }
-    $(".all-skills").on("click", ".progress-bar__add-button", function () {
-      const skillNr = this.id.replace('add', '');
-      update_exp(skillNr);
-    });
-    $(".all-skills").on("keyup", ".progress-bar__add-input", function (event) {
-      if (event.keyCode === 13) {
-        const skillNr = this.id.replace('addVal', '');
-        update_exp(skillNr);
-      }
-    });
-    $(".all-skills").on("click", ".skill__remove", function () {
-      const skillNr = this.id.replace('remove', '');
-      LifeGamification.models.removeSkill(LifeGamification.skillsView[skillNr])
-        .then(LifeGamification.main.resetView);
-    });
-  }
 
-  //End of functions to both Home and Edit views.
+    _createInput(pg_buttons) {
+      const input = this._createElement('input', {
+          "class": "progress-bar__add-input",
+          "type": "number",
+          "value": "1"
+        }, '', pg_buttons
+      );
+      input.onkeyup = (event) => {
+        if (event.keyCode === 13) {
+          this._addSkillExp(input);
+        }
+      };
 
-  const skillHomeHTML = function (number) {
-    return (`<div class="skill">`) + LifeGamification.home.skillHTML(number);
-  }
+      const addButton = this._createElement(
+        'span', {"class": "progress-bar__add-button"}, "+", pg_buttons);
 
-  const appendHomeSkill = function (skill) {
-    $('.all-skills').append(skillHomeHTML(LifeGamification.skillsView.length));
-    LifeGamification.skillsView.push(skill);
-    LifeGamification.home.viewLevelAndExp(skill);
+      addButton.onclick = () => {
```

Comment:
If we are adding event listeners it could be better to hold that convention and change `onclick` to `.addEventListener`.